src/functionaltests/java/com/ericsson/ei/utils/TestConfigs.java

```diff
@@ -76,6 +78,17 @@ MongoClient mongoClient() throws IOException {
         return null;
     }
 
+    @Bean
+    void setAuthorization() {
+        String password = StringUtils.newStringUtf8(Base64.encodeBase64("password".getBytes()));
+        System.setProperty("ldap.enabled", "true");
+        System.setProperty("ldap.url", "ldap://ldap.forumsys.com:389/dc=example,dc=com");
+        System.setProperty("ldap.base.dn", "dc=example,dc=com");
+        System.setProperty("ldap.username", "cn=read-only-admin,dc=example,dc=com");
+        System.setProperty("ldap.password", password);
+        System.setProperty("ldap.user.filter", "uid={0}");
```

Comment:
Do you really need to set all these parameters? Can't you mock the Ldap lookup/call with just Access granted return value or what valid value that ldap returns when a user get autherized ?

Reply:
For testing ldap, ldap.enabled should be set to true, If we set ldap.enabled: true, then all these parameters are required.